frontend/src/Series/Details/SeriesDetails.css

```diff
@@ -119,6 +119,14 @@
   margin: 5px 10px 5px 0;
 }
 
+.fileCountMessage {
+  font-weight: 300;
+  font-size: 15px;
+  color: #e1e2e3;
```

Comment:
I think `color: $white` works a bit better, though I think it's better in `Tooltip.css` in the `&.inverse` section:

```
&.inverse {
  background-color: $themeDarkColor;
  box-shadow: 0 5px 10px $popoverShadowInverseColor;
  color: $white;
}
```

Also looks like these are in the wrong order and failing in CI.